### PR TITLE
ConnectAd: Bidder params update

### PR DESCRIPTION
**Pull request by rtuschkany**
Accept multiple networkid and siteid types to fix https://github.com/prebid/Prebid.js/issues/6361

### CODE REVIEW

File: static/bidder-info/connectad.yaml [hunk 1]
```diff
@@ -1,16 +1,29 @@
...
+      - banner
 userSync:
-  iframe:
-    url: "https://cdn.connectad.io/connectmyusers.php?gdpr={{.GDPR}}&consent={{.GDPRConsent}}&us_privacy={{.USPrivacy}}&cb={{.RedirectURL}}"
+  redirect:
+    url: "https://sync.connectad.io/ImageSyncer?gdpr={{.GDPR}}&consent={{.GDPRConsent}}&us_privacy={{.USPrivacy}}&gpp={{.GPP}}&gpp_sid={{.GPPSID}}&cb={{.RedirectURL}}"
     userMacro: ""
     # connectad appends the user id to end of the redirect url and does not utilize a macro
+  iframe:
+    url: "https://sync.connectad.io/iFrameSyncer?gdpr={{.GDPR}}&consent={{.GDPRConsent}}&us_privacy={{.USPrivacy}}&gpp={{.GPP}}&gpp_sid={{.GPPSID}}&cb={{.RedirectURL}}"
```
Comment (gargcreation1992) [inline static/bidder-info/connectad.yaml:27]: Screenshot 2024-06-17 at 1 13 43 PM

Getting 403 for userSync. Please check.

```
https://sync.connectad.io/iFrameSyncer?gdpr=&consent=&us_privacy=&gpp=&gpp_sid=&cb=https%3A%2F%2Fib.adnxs.com%2Fprebid%2Fsetuid%3Fbidder%3Dconnectad%26gdpr%3D%26gdpr_consent%3D%26gpp%3D%26gpp_sid%3D%26f%3Db%26uid%3D
```

Comment (Sonali-More-Xandr) [reply static/bidder-info/connectad.yaml:27]: I am also getting 403

Comment (rtuschkany) [reply static/bidder-info/connectad.yaml:27]: are you trying to access that url via VPN? Where are you located? both sync urls are up and running

Comment (gargcreation1992) [reply static/bidder-info/connectad.yaml:27]: We are trying from India.

Comment (rtuschkany) [reply static/bidder-info/connectad.yaml:27]: As long as you are not calling it from a suspicious, banned, or datacenter ip, there will be no issue @gargcreation1992

**Review by Sonali-More-Xandr — APPROVED** [review]

File: static/bidder-info/connectad.yaml (hunk 1 above)
Comment (Sonali-More-Xandr) [reply static/bidder-info/connectad.yaml:27]: Checked again. Worked fine this time.

**Review by gargcreation1992 — APPROVED** [review]

**Pull request merged**